code/pizzabot.js

```diff
@@ -1,7 +1,73 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
 
-//Put your Javscript code here:
+const submitForm = document.getElementById("form-action");
+
+document.getElementById("welcomeMessage").innerHTML=(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`);
+
+/*A function that checks if userinput-choice is on menu. Returns true or false. The argument 
+is the input from the form in html and is passed as a value to the parameter "chosenPizza" in the js function. 
+Also uses the toLowerCase method to allow for users using capitals and lower case. Stores the true
+value in orderName. Is checked in if-statement: if true a particular code is run. If false, another code is run. ????*/ 
+const validateOrderName = (chosenPizza) => {
+   if (chosenPizza.toLowerCase() === vegetarian.toLowerCase() || chosenPizza.toLowerCase() === pepperoni.toLowerCase() || chosenPizza.toLowerCase() === hawaiian.toLowerCase()) {
+      return true;
+   } else {
+      return false;
+   }
+};
+
+//Function that provides us a totalprice of order by calculating price and number of pizzas. Will run of if-statement is true.
+const calculateTotalCost = (pizzaPrice, orderQuantity) => {
+   orderTotal = (pizzaPrice * orderQuantity);
+   return orderTotal;
```

Comment:
No need to parentheses in `(pizzaPrice * orderQuantity)`. For mathematical operations we should use them in same cases as in mathematics from school, so only when we want to change order of calculations (+ or - in connection with * or/ ).
I would simplify this code in other way as well : content of you function can just be :
`return pizzaPrice * orderQuantity`.
And since it's only one line of content, we can simplify it even more :
const calculateTotalCost = (pizzaPrice, orderQuantity) => pizzaPrice * orderQuantity;
We can do that because in arrow functions we can omit curly brackets (`{}`) and `return` keyword if we only have one line of code inside.
For further reference on that subject, please check out https://www.w3schools.com/js/js_arrow_function.asp